server.js

```diff
@@ -21,9 +21,60 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  const WelcomePage = {
+    " Hello music lovers": " This is an open API for top music albums.",
+    Routes: [
+      {
+        "/songs": "Get listed songs.",
+        "/songs/trackName/:trackName": "Lists the specific song track",
+        "//songs/artistName/:artistName": "Lists the specific artist name ",
+        "/endpoints": "To get API endpoints",
+      },
+    ],
+  };
+  res.send(WelcomePage);
 });
 
+app.get("/songs", (req, res) => {
+  res.status(200).json({
+    data: topMusicData,
+    success: true,
+  });
+});
+app.get("/songs/trackName/:trackName", (req, res) => {
+  const trackname = topMusicData.find(
+    (data) =>
+      data.trackName.toLocaleLowerCase() ===
+      req.params.trackName.toLocaleLowerCase()
+  );
+  if (!trackname) {
+    res.status(404).json({
+      data: "not found",
+      success: false,
```

Comment:
This worked when I tried it out! A tip could be to write something more specific like "Trackname not found" :)